cloudfront.tf

```diff
@@ -62,6 +62,22 @@ resource "aws_cloudfront_distribution" "website" {
     prefix          = "${var.hostname}/cloudfront"
   }
 
+  price_class = "PriceClass_100"
```

Comment:
I'm going to remove the addition here `price_class` as it overrides the version available above, and in turn hard-codes support to `PriceClass_100` when the above version of configurable.